_includes/head.html

```diff
@@ -15,6 +15,7 @@
     <meta property="og:description" content="{{ site.description }}" />
     <meta property="og:title" content="{{ page.title }}" />
 
+    <style>@import url('https://fonts.googleapis.com/css2?family=Fira+Sans&display=swap');</style>
```

Comment:
whats the reason for doing @import over a standard link? Wouldn't the <link> be better?

Reply:
Fixed :)